### DISCUSSION TITLE
Welcome to KTH-IV1201-Recruitment-Application Discussions!

### DISCUSSION THREAD

**Discussion by Hawkrin** [Hawkrin/KTH-IV1201-Recruitment-Application#73] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Hawkrin** [comment]
Testing

**Reply by MurtadhaAlobaidi** [reply]
Discussions test1 Done 👍

**Reply by MurtadhaAlobaidi** [reply]
🔢 👍

**Comment by Hawkrin** [comment]
Testing2

**Reply by MurtadhaAlobaidi** [reply]
💯 👍

**Comment by MurtadhaAlobaidi** [comment]
Testing3 ;)

**Reply by Hawkrin** [reply]
Nice bror

**Reply by MurtadhaAlobaidi** [reply]
👍 🥇

**Reply by MurtadhaAlobaidi** [reply]
👍

**Comment by MurtadhaAlobaidi** [comment]
Testing 4

**Reply by Hawkrin** [reply]
niice

**Reply by MurtadhaAlobaidi** [reply]
First test done
